docs/reference/commandline/create.md

```diff
@@ -68,7 +68,11 @@ Creates a new container.
       -u, --user=""                 Username or UID
       --ulimit=[]                   Ulimit options
       --uts=""                      UTS namespace to use
-      -v, --volume=[]               Bind mount a volume
+      -v, --volume=[]               Bind mount a volume with: [host-dir:]container-dir[:<options>], where
```

Comment:
@thaJeztah seems that volume-name was not mentioned by other parts.

Ok if you mean we add that , do you think we need explain what's the volume-name looks like?

I am not sure if it is just plain-string or something else. Any official reference for that ?